docs/Manuals/Server/options.rst

```diff
@@ -0,0 +1,1176 @@
+
+``tilesperplayer``
+  :strong:`Value (Min, Max)`: 100 (1, 1000)
+
+  :strong:`Description`: Number of (land) tiles per player. This value is used to determine the map
+  dimensions. It calculates the map size at game start based on the number of players and the value of
+  the setting ``landmass``. For this option to take effect, the "Map size definition" option (``mapsize``)
+  must be set to "Tiles per player" (``PLAYER``).
+
+``timeaddenemymove``
+  :strong:`Value (Min, Max)`: 0 (0, 8639999)
+
+  :strong:`Description`: Timeout at least :math:`n` seconds when enemy moved. Any time a unit moves while in
+  sight of an enemy player, the remaining timeout is increased to this value. This setting helps with
+  :term:`RTS`.
+
+``timeout``
+  :strong:`Value (Min, Max)`: 0 (-1, 8639999)
+
+  :strong:`Description`: Maximum seconds per turn. If all players have not hit :guilabel:`Turn Done` before
+  this time is up, then the turn ends automatically. Zero means there is no timeout. In servers compiled with
+  debugging, a timeout of -1 sets the autogame test mode. Only connections with hack level access may set the
+  timeout to fewer than 30 seconds. Use this with the command ``timeoutincrease`` to have a dynamic timer. The
+  first turn is treated as a special case and is controlled by the ``first_timeout`` setting.
+
+``tinyisles``
+  :strong:`Value (Min, Max)`: disabled
+
+  :strong:`Description`: Presence of 1x1 islands. This setting controls whether the map generator is allowed
+  to make islands of one only tile size.
+
+``topology``
+  :strong:`Value (Min, Max)`: Wrap East-West and Isometric (``WRAPX|ISO``)
+
+  :strong:`Description`: Freeciv21 maps are always two-dimensional. They may wrap at the north-south and
+  east-west directions to form a flat map, a cylinder, or a torus (donut). Individual tiles may be rectangular
+  or hexagonal, with either an overhead ("classic") or isometric alignment. To play with a particular
+  topology, clients will need a matching tileset.
+
+  Possible values (option can take any number of these):
+
+    * ``WRAPX``: Wrap East-West
+    * ``WRAPY``: Wrap North-South
+    * ``ISO``: Isometric
+    * ``HEX``: Hexagonal
+
+``trade_revenue_style``
+  :strong:`Value (Min, Max)`: Classic Freeciv (``CLASSIC``)
+
+  :strong:`Description`: The formula for the trade a city receives from a traderoute. ``CLASSIC`` revenues
+  depend on distance and trade with multipliers for overseas and international routes. ``SIMPLE`` revenues are
+  proportional to the average trade of the two cities.
+
+``trademindist``
+  :strong:`Value (Min, Max)`: 9 (1, 999)
+
+  :strong:`Description`: Minimum distance (tiles) for trade routes. In order for two cities in the same
+  civilization to establish a trade route, they must be at least this far apart on the map. For square grids,
+  the distance is calculated as *Manhattan distance*, that is, the sum of the displacements along the
+  :math:`x` and :math:`y` directions.
```

Comment:
This should probably mention hexagonal grids as well, since it mentions square grids

Reply:
is the math different for hex?